packages/medusa/src/api/routes/admin/discounts/create-condition.ts

```diff
@@ -0,0 +1,100 @@
+import { IsOptional, IsString } from "class-validator"
```

Comment:
this file is also added in [that PR](https://github.com/medusajs/medusa/pull/1355/files#diff-c660cf34811108809d8751be579710b20bf10ec78068da7f90c39bf469875d7b) is that normal?

Reply:
I branched from Oli's PR to get the request classes since the API at the time when i branched was established and the discussion at that point was mainly with the services and contents of the endpoint